common/common.cpp

```diff
@@ -1007,7 +1019,17 @@ void gpt_print_usage(int /*argc*/, char ** argv, const gpt_params & params) {
     printf("                        fraction of the model to offload to each GPU, comma-separated list of proportions, e.g. 3,1\n");
     printf("  -mg i, --main-gpu i   the GPU to use for the model (with split-mode = none),\n");
     printf("                        or for intermediate results and KV (with split-mode = row) (default: %d)\n", params.main_gpu);
-#endif
+#endif // LLAMA_SUPPORTS_GPU_OFFLOAD
+#ifdef GGML_USE_CLBLAS
+    printf("  -nommq, --no-mul-mat-q\n");
+    printf("                        use " GGML_CUBLAS_NAME " instead of custom mul_mat_q " GGML_CUDA_NAME " kernels.\n");
+    printf("                        Not recommended since this is both slower and uses more VRAM.\n");
+#endif // GGML_USE_CLBLAS
+#ifdef GGML_USE_SYCL
+    printf("  -nommq, --no-mul-mat-q\n");
+    printf("                        use " GGML_SYCL_NAME " instead of custom mul_mat_q " GGML_SYCL_NAME " kernels.\n");
+    printf("                        Not recommended since this is both slower and uses more VRAM.\n");
+#endif // GGML_USE_SYCL
```

Comment:
The `-nommq` parameter has been removed from the command line help since it doesn't do anything, don't reintroduce it here.

Reply:
removed in https://github.com/ggerganov/llama.cpp/pull/2690/commits/fb15de38efdecab93d73cb52d413d32ea2f38924